src/mono/browser/runtime/driver.c

```diff
@@ -253,6 +253,35 @@ mono_wasm_invoke_method (MonoMethod *method, void* args)
 	MONO_EXIT_GC_UNSAFE;
 }
 
+#ifndef DISABLE_THREADS
+
+extern void mono_threads_wasm_async_run_in_target_thread_vii (void* target_thread, void (*func) (gpointer, gpointer), gpointer user_data1, gpointer user_data2);
+extern void mono_threads_wasm_sync_run_in_target_thread_vii (void* target_thread, void (*func) (gpointer, gpointer), gpointer user_data1, gpointer user_data2);
+
+static void
+mono_wasm_invoke_jsexport_async_post_cb (MonoMethod *method, void* args)
+{
+	mono_wasm_invoke_jsexport (method, args);
+	// TODO assert receiver_should_free ?
+	free (args);
```

Comment:
mono_wasm_invoke_jsexport has a `args ? invoke_args : NULL` which means `args` can be `NULL`, but we don't have a null check here so we could try to `free(NULL)` and crash.